openrtb_ext/imp_openx.go

```diff
@@ -1,10 +1,12 @@
 package openrtb_ext
 
+import "encoding/json"
+
 // ExtImpOpenx defines the contract for bidrequest.imp[i].ext.prebid.bidder.openx
 type ExtImpOpenx struct {
-	Unit         string                 `json:"unit"`
+	Unit         json.Number            `json:"unit"`
 	Platform     string                 `json:"platform"`
 	DelDomain    string                 `json:"delDomain"`
-	CustomFloor  float64                `json:"customFloor"`
+	CustomFloor  json.Number            `json:"customFloor"`
```

Comment:
@laurb9 
could use `StringInt` data type for Unit, CustomFloor
https://github.com/prebid/prebid-server/blob/a1f48b54af8e831927a0f6838f56493fa03efb08/util/jsonutil/stringInt.go#L7-L29